config/locales/client.en.yml

```diff
@@ -121,6 +121,9 @@ en:
 
       composer:
         toggle_toolbar: "Toggle toolbar"
+        italic_text: "emphasized text"
+        bold_title: "Strong"
+        code_text: "indent preformatted text by 4 spaces"
```

Comment:
Sentence case or lower case? Either way, we should stick to one 😉

Reply:
I copied and pasted the wrong key :joy: should have gotten `bold_text`, fixing